BaragonService/src/main/java/com/hubspot/baragon/service/worker/BaragonRequestWorker.java

```diff
@@ -302,6 +304,31 @@ public void run() {
     }
   }
 
+  private QueuedRequestWithState setNoValidateIfPossible(QueuedRequestWithState nonServiceChangeRequest) {
+    BaragonRequest originalRequest = nonServiceChangeRequest.getRequest();
+    if (originalRequest.getAddUpstreams().isEmpty() && originalRequest.getReplaceUpstreams().isEmpty() && !originalRequest.getRemoveUpstreams().isEmpty()) {
```

Comment:
nit, but can we make this a nicely named method. Read the if statement 3 times before realizing there was a ! on the remove, lol. Maybe `isOnlyUpstreamRemoval()` or something?